python/vineyard/contrib/ml/xgboost.py

```diff
@@ -19,6 +19,7 @@
 from vineyard._C import ObjectMeta
 from vineyard.data.utils import from_json, to_json, build_numpy_buffer, normalize_dtype
 from vineyard.data import tensor, dataframe, arrow
+from vineyard.core.resolver import resolver_context
```

Comment:
Please put the `from vineyard.core.resolver import ....` before the line `from vineyard.data.utils import ...` to maintain the import order.